dlp/snippets/inspect_content.py

```diff
@@ -983,6 +985,67 @@ def inspect_image_file_all_infotypes(
 # [END dlp_inspect_image_all_infotypes]
 
 
+# [START dlp_inspect_image_listed_infotypes]
+def inspect_image_file_listed_infotypes(
+    project: str,
+    filename: str,
+    info_types: List[str],
+    include_quote=True,
+) -> None:
+    """Uses the Data Loss Prevention API to analyze strings for protected data
```

Comment:
"analyze strings in an image for data matching the given infoTypes"

(also infoTypes is one word in our documentation, so lets be consistent here).

Reply:
updated, Thank you for reviewing